static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -79,17 +82,22 @@ export function InviteBanner({
     }).then(prompt => {
       setShowBanner(!promptIsDismissed(prompt));
     });
-  }, [api, organization, promptsFeature, hideBanner]);
+  }, [api, organization, promptsFeature, isEligibleForBanner]);
 
   useEffect(() => {
     const {inviteMissingMembers} = qs.parse(location.search);
 
-    if (!hideBanner && inviteMissingMembers) {
+    if (!isEligibleForBanner && inviteMissingMembers) {
```

Comment:
this should just be `isEligibleForBaner && inviteMissingMembers` right?

Reply:
yes